website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,473 @@
+---
+title: Customize schema and alias
+id: customize-schema-alias
+description: "How to properly adjust your generate_schema_name() and generate_alias_name() macros.."
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
```

Comment:
might be helpful to have an intro sentence so readers understand the tldr of this article/guide
```suggestion
This guide explains how to customize the [schema](/docs/build/custom-schemas) and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
```